tests/plugins/tasks/networking/test_netmiko_file_transfer.py

```diff
@@ -1,9 +1,13 @@
+import os
+
 from brigade.plugins.tasks import networking
 
+THIS_DIR = os.path.dirname(os.path.realpath(__file__))
+
 
 class Test(object):
     def test_netmiko_file_transfer(self, brigade):
-        source_file = 'test_file.txt'
+        source_file = os.path.join(THIS_DIR, 'data/test_file.txt')
```

Comment:
you should probably do `os.path.join(THIS_DIR, 'data', 'test_file.txt')`. For consistency mostly :P (not that we are very good at it anyway :(

Reply:
Sure. The contribution guidelines never mentioned this. ;)